### DISCUSSION TITLE
Perform rank commands on rank down

### DISCUSSION THREAD

**Discussion by ghost** [Zrips/CMI#4652] [Feature Requests]
Please add the option for commands to run when a player ranks down to a specific rank. I want commands to run for a rank on the rank that the player ranks down to.

Commands can run when a player ranks up to the rank, and commands can run from the rank when the player ranks down.

So for example if I want each rank to give a player access to a specific kit. The player is at rank 3 and has access to the rank 3 kit. The player then ranks up to 4. The commands run when the player ranks up to 4 to remove the rank 3 kit permission and give them the rank 4 kit permission.
If the player ranks down from 4 to 2, I currently can only remove the rank 4 kit permission, but I cannot give them the rank 2 kit permission from that action because the commands for rank 2 won't run unless they are ranking up.

So if you can add a setting that we can put on ranks like "RunCommandsOnRankDown" to make the configured rank commands run whenever the player ranks down to that rank, that would be good.

**Comment by TheGamersBrew** [comment]
Honestly without this I have to find another plugin to take it's place and disable Ranks in CMI. I agree this is in EssentialX so idk why this isn't already in this plugin.
 
a Command as simple as 
/cmi rankforce up. 

That said adding a parameter for grouping a single rank tree together would go a long way too. 

/cmi rankforce up testranks

**Reply by mrfloris** [reply]
To be fair, just because something's in another plugin doesn't automatically mean it is or has to be in CMI as well. That said, it would be nice if we can run commands on rankup and rankdown. Yeh
